### ISSUE TITLE
Pretraining code of Label-aware MLM

### ISSUE THREAD

**Issue by GillesJ** [thu-coai/SentiLARE#3]
I have read your paper and found it the label-aware masked-language model pretraining with SentiWordNet integration very interesting. I am curious to see how it is implemented practically for my research.

I was wondering if you had any plans to publish the code of this part of the research?

**Comment by kepei1106** [comment]
Hi, sorry for the late reply. We have released the pre-training codes. Feel free to contact us if you have other questions.
